thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/db/dao/AnomalyTaskDAO.java

```diff
@@ -63,7 +63,8 @@ public boolean updateStatusAndWorkerId(Long workerId, Long id, TaskStatus oldSta
         return true;
       }
     } catch (Exception e) {
-      LOG.info("OptimisticLockException");
+      LOG.info("OptimisticLockException while updating workerId {} and status {} for anomalyTaskId {}",
+          workerId, newStatus, id);
```

Comment:
Would be good to preserve the base exception in the log. 
you can do this by passing exception e in the end as param.

log.info(message, messageObjects.., throwable)